src/server.lisp

```diff
@@ -47,6 +47,10 @@
 
 (in-package #:rpcq)
 
+(defstruct server-auth-config
+  "Holds the ZeroMQ Curve configuration for a server socket."
+  (server-secret-key nil :type (or null string))
```

Comment:
can these really be null? can they be RO?

Reply:
Likewise no, will switch to RO and required.

Actually the sever public key is unused. Only clients need it. I was sort just mimicking the `ServerAuthConfig` on the python side. Maybe I should drop the `server-public-key` slot, but then it feels gratuitous to define a struct with a single slot..., but nice to have and abstract type in case we ever want to add client authentication, in which case this can also hold the client keys or path to a directory where to find them, etc... will mull this one over.

Reply:
Mulling complete. Opted to just drop it since it's unused and who wants unused cruft gunking up their structs?